### PR TITLE
Update Snaps code samples

### LINKED ISSUE #1144: Update Snaps code samples
Update Snaps code samples to match guidelines used for Wallet code samples. See https://github.com/MetaMask/metamask-docs/pull/1116 for guidelines.

### PR DESCRIPTION
**Pull request by alexandratran**
Update code samples:

- Prefer double quotes over single quotes
- Prefer four-space indentation over two-space indentation
- Prefer trailing commas for ts and js files
- Add titles with file names where applicable
- Make code comments consistent

Fixes #1144

### CODE REVIEW

File: snaps/features/custom-evm-accounts/create-account-snap.md [hunk 1]
```diff
@@ -155,11 +155,11 @@ to MetaMask and your dapp:
...
 ```typescript
 export const onKeyringRequest: OnKeyringRequestHandler = async ({
-  origin,
-  request,
+    origin,
+    request,
 }) => {
-  // Your custom logic here...
-  return handleKeyringRequest(keyring, request);
+    // Your custom logic here.
```
Comment (joaniefromtheblock) [inline snaps/features/custom-evm-accounts/create-account-snap.md:161]: Just to make it consistent with the other comments

```suggestion
    // Add custom logic here.
```

File: snaps/features/custom-evm-accounts/security.md [hunk 2]
```diff
@@ -28,20 +28,20 @@ For example:
...
-        EthMethod.Sign,
-        EthMethod.SignTransaction,
-        EthMethod.SignTypedDataV1,
-        EthMethod.SignTypedDataV3,
-        EthMethod.SignTypedDataV4,
-      ],
-      type: EthAccountType.Eoa,
+        id: uuid(),
+        options: {
+            privateKey: "0x01234...78", // !!! DO NOT DO THIS !!!
```
Comment (joaniefromtheblock) [inline snaps/features/custom-evm-accounts/security.md:33]: Maybe more descriptive comment in addition to the  "DO NOT DO THIS"

Comment (alexandratran) [reply snaps/features/custom-evm-accounts/security.md:33]: The text preceding the code sample provides context on what not to do, I think this comment is just highlighting the line where it applies: https://docs.metamask.io/snaps/features/custom-evm-accounts/security/#do-not-add-secret-information-to-account-objects

**Review by joaniefromtheblock — APPROVED** [review]
LGTM

**Pull request merged**